service/src/main/kotlin/fi/espoo/evaka/children/ChildControllerCitizen.kt

```diff
@@ -78,21 +83,61 @@ class ChildControllerCitizen(private val accessControl: AccessControl) {
         val defaultServiceNeedOptions =
             tx.getServiceNeedOptions()
                 .filter { it.defaultOption }
-                .associateBy({ it.validPlacementType }, { ServiceNeedOptionPublicInfo.of(it) })
+                .associateBy { it.validPlacementType }
         val serviceNeedDateRanges = serviceNeeds.map { FiniteDateRange(it.startDate, it.endDate) }
         return tx.getPlacementSummary(childId).flatMap { placement ->
             val placementRange = FiniteDateRange(placement.startDate, placement.endDate)
             placementRange.complement(serviceNeedDateRanges).map {
+                val defaultServiceNeedOption = defaultServiceNeedOptions[placement.type]
                 ServiceNeedSummary(
                     it.start,
                     it.end,
-                    defaultServiceNeedOptions[placement.type],
+                    defaultServiceNeedOption?.let { sno -> ServiceNeedOptionPublicInfo.of(sno) },
+                    defaultServiceNeedOption?.contractDaysPerMonth,
                     placement.unit.name
                 )
             }
         }
     }
 
+    @GetMapping("/{childId}/attendance-summary/{yearMonth}")
```

Comment:
I would add a simple integration test which calls this endpoint to at least smoke test the sql queries